packages/SwingSet/test/test-comms.js

```diff
@@ -67,18 +68,25 @@ function mockSyscall() {
   const sends = [];
   const resolves = [];
   const gcs = [];
+  const subscribes = [];
   const fakestore = new Map();
   const syscall = harden({
-    send(targetSlot, method, args, _result) {
-      sends.push([targetSlot, method, args]);
+    send(targetSlot, method, args, result) {
+      if (result) {
+        sends.push([targetSlot, method, args, result]);
+      } else {
+        sends.push([targetSlot, method, args]);
+      }
       // return 'r-1';
     },
```

Comment:
Out of curiosity, why not do 
```suggestion
    send(...sendArgs) {
      sends.push(sendArgs);
      // return 'r-1';
    },
```

Reply:
That might work. I know we've had problems with `undefined` vs `null` in the past, so I've learned to be very explicit with the lengths of arrays that we send over the syscall wire, but this (test) case probably isn't vulerable to those problems.

Reply:
Oh, so I applied that patch and now the test is failing https://github.com/Agoric/agoric-sdk/runs/6278370102?check_suite_focus=true#step:5:1390 . It looks like `sends` is being given a four-element Array (with an `undefined` at the end). So maybe `send()` is being called with an `undefined` at the end, but `...sendArgs` is paying attention to the strict arity of the call site, rather than the defined-ness of the values.